definitions/npm/bluebird_v3.x.x/flow_v0.104.x-/test_bluebird_v3.x.x.js

```diff
@@ -78,10 +78,9 @@ fooPromise(1, 'b').catch(function(e) {
 fooPromise(1, 'b').catch(Error, function(e: Error) {
   let m: string = e.message;
 });
-// $FlowExpectedError
-fooPromise(1, 'b').catch(Error, function(e: NetworkError) {
```

Comment:
I believe this wasn' tthe right test: `NetworkError` doesn't exist and this was failing for the wrong cause.